platforms/documentation/docs/build.gradle.kts

```diff
@@ -17,6 +18,23 @@ plugins {
     id("org.asciidoctor.jvm.convert")
     id("gradlebuild.documentation")
     id("gradlebuild.generate-samples")
+    id("gradlebuild.android-home-warmup")
+}
+
+androidHomeWarmup {
+    sdkVersions.set(
+        listOf(
+            // Used by declaringConfigurations-kmp (AGP 8.11.2) and declaringConfigurations-android (AGP 8.13.0)
```

Comment:
This should be aligned with the samples, but how to remember updating them after making changes to the samples?

Reply:
@lkasso Is there a process that get's Android samples updated to the latest AGP version?

Judging from the list below I assume there is none.

Reply:
No, there is no process.